src/main/java/de/uni_tuebingen/ub/nppm/db/SaveHelper.java

```diff
@@ -67,7 +121,7 @@ public static void updateAttribute(String table, String attribute, String value,
         updateAttribute(table, attribute, value, Integer.parseInt(id));
     }
 
-     public static void updateAttribute(String table, String attribute, String value, Map<String, String> andConditions) throws Exception {
+    public static void updateAttribute(String table, String attribute, String value, Map<String, String> andConditions) throws Exception {
```

Comment:
By the way, it seems like the word `attribute` is overused a lot. In my understanding, you want to update one or more existing rows and set a column to a specific value using certain conditions. So  a properly named function should be something like this:

`public static void update(String table, String column, String value, Map<String,String> andConditions) throws Exception`

Maybe it would be efficient to overload this and create main implementation:
`public static void update(String table, Map<String,String> columnsAndValues, Map<String,String> andConditions)`

And then changing the existing function just calling the other one and creating a map out of the single given column + value.

Reply:
done! look at commit: update and getAttribute Method
Instead of 3 methods, I now have one and adapted it in the dosave. I also had to expand the public static List<Map<String, String>> getAttribute(String table, String attribute, Map<String, String> andConditions) because otherwise it would be in dosave (combined) wouldn't work otherwise.